packages/fxa-content-server/app/scripts/lib/auth-errors.js

```diff
@@ -586,10 +586,13 @@ var ERRORS = {
     errno: 1062,
     message: t('Invalid redirect'),
   },
+  /*
+  Removed for https://github.com/mozilla/fxa/issues/2072
```

Comment:
Nit: might as well stick with the syntax on the item above (line 579), `Removed in https://github.com/mozilla/fxa/pull/2658`

Reply:
PR was not created at the time of writing; but can do now.  👍